### PR TITLE
feat(cactus-web): Add combobox functionality to select

### PR DESCRIPTION
**Pull request by Dhalton**
Adds the option to render a select as a combobox

### CODE REVIEW

File: examples/mock-ebpp/tests/form-elements.test.ts [hunk 1]
```diff
@@ -49,39 +49,73 @@ describe('UI Config Form', () => {
     await server.close()
   })
 
-  it('should be titled "UI Config"', async () => {
+  test('should be testled "UI Config"', async () => {
```
Comment (JamesNimlos) [inline examples/mock-ebpp/tests/form-elements.test.ts]: lol, "testled"

File: examples/mock-ebpp/tests/helpers/form-actions.ts [hunk 2]
```diff
@@ -1,49 +1,99 @@
...
+    if (typeof optionOrOptions === 'string') {
+      const option = await getByText(listbox, optionOrOptions)
+      await option.tap()
+    } else if (Array.isArray(optionOrOptions)) {
+      for (let i = 0; i < optionOrOptions.length; i++) {
+        const option = await getByText(listbox, optionOrOptions[i])
+        await option.tap()
+      }
+      // haxxors
+      await this.page.keyboard.press('Escape')
```
Comment (JamesNimlos) [inline examples/mock-ebpp/tests/helpers/form-actions.ts]: Maybe we should be clicking our own "Done" button instead actually?

**Comment by JamesNimlos** [comment]
That last commit could actually be referenced as a fix in cactus-web since the dialog should have had that role all along

**Comment by Dhalton** [comment]
> That last commit could actually be referenced as a fix in cactus-web since the dialog should have had that role all along

You're right, let me reset the commit and separate them

**Pull request merged**